### DISCUSSION TITLE
ZG-003-RF

### DISCUSSION THREAD

**Discussion by Rex100J** [Koenkk/zigbee2mqtt#16329] [Devices]
I have a four channel zigbee relay ZG-003-RF wich has dry relay contacts. 
https://www.aliexpress.com/item/1005004265807787.html?gatewayAdapt=glo2nld

Domoticz zigbee2mqtt creates switch device and "power on mode" for all channels + one device for link quality. 
Switch device works ok, but "power on mode" does not.   
There is an additional button for switching mode, which would also nice to be ablt to control over Domoticz. 
Any idea what to do ?

**Comment by Rex100J** [comment]
The error I get when trying to change "power on mode" is: 

 _Error: Zigbee2MQTT: A Zigbee publish error occured for device '0xa4c138414ac186f1' with error message: Publish 'set' 'power_on_behavior' to '0xa4c138414ac186f1' failed: 'TypeError: Cannot read properties of null (reading 'toLowerCase')'_

**Comment by canedje** [comment]
I did also order such a device.
Nice to know Z2M is recognizing it.
Indeed adding the switching mode button would be nice if possible.
I only do not know : the "power on mode" what is this mode doing?

**Comment by redpixy** [comment]
does the interlock mode work?

**Comment by Rex100J** [comment]
This old problem seems not being solved. I would like to be able to set all four switches to "on" or "off" or "previous" mode on power up. But I can select only "off". Any ideas what can I do ?
